include/souper/Infer/AliveDriver.h

```diff
@@ -29,7 +29,7 @@ namespace souper {
 class AliveDriver {
   typedef std::unordered_map<const Inst *, IR::Value *> Cache;
 public:
-  AliveDriver(Inst *LHS_, Inst *PreCondition_, InstContext &IC_);
+    AliveDriver(Inst *LHS_, Inst *PreCondition_, InstContext &IC_, std::vector<Inst *> ExtraInputs = {});
```

Comment:
is the whitespace change intentional?

Reply:
No, new IDE version seems to have reset my settings and wants to force 4-spaces. :(